code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -450,34 +450,33 @@ nobiliumsuppression = INFINITY
 			air.temperature = max(((temperature * old_heat_capacity + energy_produced) / new_heat_capacity), TCMB) //the air heats up when reacting
 		return REACTING
 
-/datum/gas_reaction/nitrylformation //The formation of nitryl. Endothermic. Requires bz.
+/datum/gas_reaction/nitryum_formation //The formation of nitryum. Endothermic. Requires bz.
 	priority = 3
-	name = "Nitryl formation"
-	id = "nitrylformation"
+	name = "Nitryum formation"
+	id = "nitryum_formation"
 
-/datum/gas_reaction/nitrylformation/init_reqs()
+/datum/gas_reaction/nitryum_formation/init_reqs()
 	min_requirements = list(
-		/datum/gas/oxygen = 10,
+		/datum/gas/tritium = 20,
 		/datum/gas/nitrogen = 10,
-		/datum/gas/bz = 5,
-		"TEMP" = 1500,
-		"MAX_TEMP" = 10000
+		/datum/gas/bz = 10,
+		"TEMP" = FIRE_MINIMUM_TEMPERATURE_TO_EXIST + 100
 	)
 
-/datum/gas_reaction/nitrylformation/react(datum/gas_mixture/air)
+/datum/gas_reaction/nitryum_formation/react(datum/gas_mixture/air)
 	var/list/cached_gases = air.gases
 	var/temperature = air.temperature
 
 	var/old_heat_capacity = air.heat_capacity()
 	var/heat_efficency = min(temperature / (FIRE_MINIMUM_TEMPERATURE_TO_EXIST * 8), cached_gases[/datum/gas/oxygen][MOLES], cached_gases[/datum/gas/nitrogen][MOLES])
```

Comment:
I think you forgot to switch this over to check trit instead of O2. Also, this doesn't limit nitryum formation based on BZ so it can technically use more BZ than there is BZ in the air.

Reply:
Also, I'm pretty sure this will spam runtimes if you try to do this reaction anywhere that doesn't have oxygen.

Reply:
Ah damn thanks I didn't change that var

I'll make Bz to be just a catalyst